qa-tests-backend/src/test/groovy/UpgradesTest.groovy

```diff
@@ -17,10 +17,11 @@ import spock.lang.Tag
 import spock.lang.Unroll
 import spock.lang.IgnoreIf
 
```

Comment:
I'd prefer to leave this as is. `Env.mustGet("UPGRADE_CLUSTER_ID")` ensures that `UPGRADE_CLUSTER_ID` is set for this test and if not it would silently pass. 

In CI we ensure this test is skipped with [excludeTags](https://github.com/stackrox/stackrox/blob/172c463308606e6f4dde97c8e569361337e93f1d/qa-tests-backend/build.gradle#L159). Would something similar work for P/Z?

Reply:
Reverted the workaround for P/Z